client/acl.go

```diff
@@ -10,18 +11,38 @@ import (
 
 //https://docs.splunk.com/Documentation/Splunk/8.0.4/RESTUM/RESTusing#Access_Control_List
 func (client *Client) GetAcl(owner, app, name string, resources ...string) (*http.Response, error) {
-	resourcePath := []string{"services", owner, app}
+	resourcePath := []string{"servicesNS", owner, app}
 	resourcePath = append(resourcePath, resources...)
 	resourcePath = append(resourcePath, name, "acl")
 	endpoint := client.BuildSplunkURL(nil, resourcePath...)
 	resp, err := client.Get(endpoint)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("GET failed for endpoint %s: %s", endpoint.Path, err)
```

Comment:
I changed this while troubleshooting some failures, but I think more detailed errors will be helpful.